runtime/mainnet/src/config/proxy.rs

```diff
@@ -1,11 +1,9 @@
 use frame_support::traits::InstanceFilter;
-use pop_runtime_common::proxy::{
-	AnnouncementDepositBase, AnnouncementDepositFactor, MaxPending, MaxProxies, ProxyDepositBase,
-	ProxyDepositFactor, ProxyType,
-};
-use sp_runtime::traits::BlakeTwo256;
+use pop_runtime_common::proxy::{MaxPending, MaxProxies, ProxyType};
 
-use crate::{Balances, Runtime, RuntimeCall, RuntimeEvent};
+use crate::{
+	deposit, parameter_types, Balance, Balances, BlakeTwo256, Runtime, RuntimeCall, RuntimeEvent,
+};
 
 impl InstanceFilter<RuntimeCall> for ProxyType {
```

Comment:
this is the closest I can leave a comment.

For `ProxyType::Assets`, we don't have the assets pallet but note how in AH it uses a RuntimeCall::Assets

https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs#L494

Anything we need to do about this?

Reply:
The calls for the assets pallets could be extended to the type later on as the pallets are included in the runtime.

Although I'm happy to remove these types for now as they don't make much sense until the pallets are there. Even if it is just a naming detail.

Reply:
[d3bfc6c](https://github.com/r0gue-io/pop-node/pull/441/commits/d3bfc6c748aa46e487980649bfaaa9455ffc70b0)